exampleSite/content/docs/advanced/customization.md

```diff
@@ -10,14 +10,28 @@ This page describes the available options and how to customize the theme further
 
 ## Custom CSS
 
-To add custom CSS, we need to create a file `assets/css/custom.css` in our site. Hextra will automatically load this file. For example, customize the font family of the content:
+To add custom CSS, we need to create a file `assets/css/custom.css` in our site. Hextra will automatically load this file.
+
+### Font Family
+
+The font family of the content can be customized using:
 
 ```css {filename="assets/css/custom.css"}
 .content {
   font-family: "Times New Roman", Times, serif;
 }
 ```
 
+### Code in Plain Text
```

Comment:
Let's use "Inline Code Element" instead of "Code in Plain Text"

https://developer.mozilla.org/en-US/docs/Web/HTML/Element/code

Reply:
ok